app/src/main/res/values/strings.xml

```diff
@@ -217,4 +213,40 @@
      When the described button is pressed, the overlay will open and the browser will be hidden underneath it -->
     <string name="hint_press_menu_to_open_overlay_a11y">Press the menu button for %1$s Home, search, settings, and more</string>
 
+    <!-- FxA Strings-->
+    <!-- Content description for a button that will prompt a user to sign in, in order to receive tabs pushed from FF
+    on another device -->
+    <string name="fxa_login_a11y">Receive tabs</string>
+    <!-- Heading for settings menu item button and CTA link on new update -->
+    <string name="fxa_settings_item_new">New! Firefox tabs on Fire TV</string>
+    <!-- Heading for settings menu item button and CTA link when signed in -->
+    <string name="fxa_settings_item_signed_in">Sent tab settings</string>
+    <!-- Heading for settings menu item button and CTA link when need to sign in again -->
+    <string name="fxa_settings_item_sign_in_again">Sent tabs - sign in again</string>
+    <!-- FxA pre-boarding header -->
+    <string name="fxa_preboarding_header">See your tabs on the big screen</string>
+    <!-- FxA pre-boarding instruction text -->
+    <string name="fxa_preboarding_instruction">Send a tab from anywhere you use Firefox and see it here. Sign in to Firefox for Fire TV to get started.</string>
```

Comment:
nit: I wonder if we should be subbing the `@string/app_name_extended_fire` instead of hardcoding, "FFTV". Can you check with Delphine what the best practice is?